compose/cli/main.py

```diff
@@ -287,14 +287,15 @@ def config(self, config_options, options):
         """
         Validate and view the Compose file.
 
-        Usage: config [options]
+        Usage: config [options] [-f KEY=VAL...]
 
         Options:
             --resolve-image-digests  Pin image tags to digests.
             -q, --quiet              Only validate the configuration, don't print
                                      anything.
             --services               Print the service names, one per line.
             --volumes                Print the volume names, one per line.
+            -f, --filter KEY=VAL     Filter services by a property (can be used multiple times)
```

Comment:
This probably deserves better documentation but I wasn't sure how to format it.

In particular, it would be good to provide values that filter takes.